hooks.lua

```diff
@@ -7,17 +7,12 @@
 
 
 
-function OnPlayerLeftClick(a_Player, a_BlockX, a_BlockY, a_BlockZ, a_BlockFace, a_Action)
+function OnPlayerBreakingBlock(a_Player, a_BlockX, a_BlockY, a_BlockZ, a_BlockFace, a_lockType, a_BlockMeta)
```

Comment:
Could you rename `a_lockType` to `a_BlockType`? Also, since it's now using `OnPlayerBreakingBlock` the code at line 29 (now 34) can be removed as the blocktype is already provided as a hook parameter.

Reply:
renamed using `git commit --amend` , hope there be no problem anymore 😂